--- gpm-1.20.0/doc/doc.gpm.in.docfix	Sat Feb 23 16:42:23 2002
+++ gpm-1.20.0/doc/doc.gpm.in	Mon Mar  4 16:30:40 2002
@@ -551,7 +551,7 @@
 .SH SEE ALSO
 .nf
 \fB mev(1)\fP        A sample client for the gpm daemon.
-\fB gpm-root(1)\fP   An handler for Control-Mouse events.
+\fB gpm-types(7)\fP  Description of mouse types supported by gpm.
 
 .fi
 The info file about `gpm', which gives more complete information and
@@ -731,6 +731,7 @@
 @item imps2
 	``IntelliMouse'' on the ps/2 port. This type can also be used for
         a generic 2-button ps/2 mouse too, since it will auto-detect the type.
+	For serial ``IntelliMouse'' devices, use the ``ms3'' decoder.
 
 @item netmouse
         Decodes the ``Genius NetMouse'' type of devices on the ps/2
@@ -1862,8 +1863,8 @@
 
 .SH SEE ALSO
 .nf
-\fB gpm(8) \fP      The mouse server
-\fB gpm-root(1) \fP An handler for Control-Mouse events.
+\fB gpm(8) \fP       The mouse server
+\fB gpm-types(7) \fP Description of mouse types supported by gpm.
 
 .fi
 The info file about `gpm', which gives more complete information and
