--- gpm-1.20.7/contrib/Makefile.in.do_not_build_it_twice~	2012-10-28 09:52:29.479816651 +0100
+++ gpm-1.20.7/contrib/Makefile.in	2012-10-28 09:52:29.498816909 +0100
@@ -11,7 +11,7 @@ include $(top_builddir)/Makefile.include
 
 all: $(filter-out %.elc,$(srcdir)/$(ELISP))
 
-install: all
+install:
 	if [ -n "$(filter-out %.elc,$(ELISP))" ]; then for i in `echo $(filter-out %.elc,$(ELISP))`; do \
 		$(INSTALL_DATA) $(srcdir)/$$i $(lispdir)/`basename $$i` ;\
 	done; fi
--- gpm-1.20.7/doc/Makefile.in.do_not_build_it_twice~	2012-10-28 09:52:29.478816638 +0100
+++ gpm-1.20.7/doc/Makefile.in	2012-10-28 09:52:29.498816909 +0100
@@ -113,7 +113,7 @@ gpmdoc.ps:	gpm.ps
 installdirs:	
 	$(MKDIR) $(man1dir) $(man7dir) $(man8dir) $(infodir)	
 
-install:	all installdirs
+install: installdirs
 	$(INSTALL_DATA) -m 644 mev.1          $(man1dir)
 	$(INSTALL_DATA) -m 644 mouse-test.1   $(man1dir)
 	$(INSTALL_DATA) -m 644 gpm-root.1     $(man1dir)
--- gpm-1.20.7/Makefile.in.do_not_build_it_twice~	2012-10-26 23:21:38.000000000 +0200
+++ gpm-1.20.7/Makefile.in	2012-10-28 09:52:29.497816895 +0100
@@ -59,7 +59,7 @@ Makefile.include: config.status $(srcdir
 
 ### INSTALL
 
-install:	check installdirs do-install
+install:	installdirs do-install
 
 install-strip:
 	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM) $(STRIP)' install
--- gpm-1.20.7/src/Makefile.in.do_not_build_it_twice~	2012-10-26 23:21:38.000000000 +0200
+++ gpm-1.20.7/src/Makefile.in	2012-10-28 09:53:21.584518148 +0100
@@ -103,7 +103,7 @@ $(DEPFILE) dep: prog/gpm-root.c
 check:	all
 installdirs:
 
-install:	check 
+install:
 	$(INSTALL_PROGRAM) gpm $(sbindir)/gpm
 	if [ -f lib/libgpm.a ] ; then \
 	$(INSTALL_DATA) -m 644 lib/libgpm.a $(libdir)/libgpm.a ; \
