--- gpm-1.99.7/gpm2/Makefile.0057~	2015-07-28 00:35:56.589765774 +0200
+++ gpm-1.99.7/gpm2/Makefile	2015-07-28 00:38:45.831805942 +0200
@@ -95,15 +95,12 @@ $(SBIN)/ccontrol util/ccontrol: config.h
 	$(LD) $(LDFLAGS) $(CCO_OBJ) -o $@
 	$(STRIP) $@
 
-install: install-dir cinit cservice ccontrol
+install: install-dir
 	@echo '*** Installing cinit ***'
-	./bin/cinit.install.binary
 
 install-miniconf:
-	./bin/cinit.install.miniconf
 
 install-dir:
-	./bin/cinit.install.dir
 
 
 ################################################################################
--- gpm-1.99.7/src/daemon/open_console.c.0057~	2008-07-24 12:33:05.000000000 +0200
+++ gpm-1.99.7/src/daemon/open_console.c	2015-07-28 00:35:56.512765756 +0200
@@ -19,10 +19,12 @@
  *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  *
  ********/
+#define _BSD_SOURCE 1 /* major() */
 
 #include <fcntl.h>              /* open and co.  */
 #include <sys/stat.h>           /* stat() */
 #include <sys/ioctl.h>          /* ioctl() */
+#include <sys/types.h>		/* major() */
 
 /* Linux specific (to be outsourced in gpm2 */
 #include <linux/serial.h>       /* for serial console check */
--- gpm-1.99.7/src/headers/daemon.h.0057~	2008-07-24 12:32:54.000000000 +0200
+++ gpm-1.99.7/src/headers/daemon.h	2015-07-28 00:35:56.512765756 +0200
@@ -121,7 +121,7 @@ extern struct mouse_features  mouse_tabl
 extern Gpm_Type         mice[];
 extern Gpm_Type         *repeated_type;
 
-time_t                  last_selection_time;
+extern time_t           last_selection_time;
 
 
 /*************************************************************************
