--- gpm-1.99.7/src/lib/report-lib.c.0050~	2015-07-27 23:36:44.555922732 +0200
+++ gpm-1.99.7/src/lib/report-lib.c	2015-07-27 23:38:28.231947338 +0200
@@ -44,8 +44,11 @@ void gpm_report(int line, char *file, in
    line = 0;                    /* FIXME: gpm 1.99.13 */
    file = NULL;
 
+#ifndef DEBUG_VERBOSE
    if(stat == GPM_STAT_DEBUG)
       return;
+#endif
+
    va_start(ap, text);
 
    switch (stat) {
